cypress/component/components/NavBar.cy.js

```diff
@@ -0,0 +1,64 @@
+import NavBar from '../../../src/components/NavBar.vue';
+
+const staticResponse = {
+  fields: {
+    claims: {
+      mapValue: {
+        fields: {
+          adminUid: { stringValue: 'yXuZ8S0En1UsOE4C0uh6wUlQ5Wt1' },
+          assessmentUid: { stringValue: 'FW0G9oXYQjMpwPdyptVJOsbb6J42' },
+          roarUid: { stringValue: 'yXuZ8S0En1UsOE4C0uh6wUlQ5Wt1' },
+          someOtherField: { stringValue: 'someOtherValue' },
+        },
+      },
+    },
+  },
+};
+
+describe('<NavBar />', () => {
+  beforeEach(() => {
+    cy.setAuthStore().as('authStore');
+
+    // Intercept network calls and respond with mock data
+    cy.intercept(
+      'GET',
+      'https://firestore.googleapis.com/v1/projects/gse-roar-admin-dev/databases/(default)/documents/userClaims/**/*',
```

Comment:
No need to do this now, but when we write further component tests, it might be good to move these to Cypress constants so that we don't repeat the string literals everywhere.

Reply:
Good idea, I have added the base url to the cypress config. Feel free to suggest different names or other variables that we should add.